### ISSUE TITLE
Complete Training Code and Guidelines

### ISSUE THREAD

**Issue by tianlinzx** [Inferencer/LipSick#23]
Would you please add complete training code and scripts and guidelines ?

**Comment by Inferencer** [comment]
Training code can be found at the repo of DINet as we use the same pre-trained model just renamed to pre-trained_lipsick.pth 
Doesn't have loss convergence charts though so will need to implement that or you will be guessing when to stop each time.

I talked about it a little in another issue
https://github.com/Inferencer/LipSick/issues/3#issuecomment-2079060760
